### DISCUSSION TITLE
Samsung S22, Dexcom one issues and solutions

### DISCUSSION THREAD

**Discussion by Lukvdv** [NightscoutFoundation/xDrip#2415] [General]
Started 3 days ago with this CGM, no dexcom support for the M22 so I installed Xdrip. Super! But a lot of unbonding problems.
After some thinking and checking all settings I could narrow the problem down to some point.

As long as the phone stays in one place, no problems, normal readings. When I move around (I live on a big farm) I had connection issues... When pressing, at system status, the restart the collector (start verzamerlaar opnieuw in dutch) I was asked to allow bt connection with the transmitter and got new readings, for some time.
 At first I thought they were caused by switching between different WiFi or network connections, so I switched off my WiFi on the phone. This made it a bit more stable but not perfect.
Digging deeper in the WiFi section I found a 'sync to Samsung cloud' and switched this off, because I wanted to use my WiFi. This way I was able to walk around without loss of connection with the transmitter. So, ok, this could be the solution!
But when the phone connected to other bt devices, same problem again.
Digging deeper in the bt section on my phone, yes, the same option, sync to Samsung cloud.
Also switched this off.

On this moment I have a stable connection with the dexcom transmitter whatever I do. 

Maybe someone can take a look at this sync to samnsung cloud function and why it causes problems with the dexcom? I am willing to do some more testing but everything works now like it should. I think this might be a solution for other people with Samsung smartphones. 
Mine is a Galaxy M22, SM-M225FV/D running Android 12

Greetings

**Comment by Navid200** [comment]
Would you please post a screenshot of the this "sync to Samsung cloud" setting so that we can see where in the menu it is?
I know it will be in Dutch.  That's OK.

**Comment by Lukvdv** [comment]
The 3 dots in first pic after ´stoppen´ go to the second menu, ´geavanceerd´.There, first item, is switched of now. Both in WiFi and bt menu. Acces to this menu when swiping top down in home screen.I also switched off one thing in Xdrip, when on it was less stable. I dont remember what exactly but I will mail this later so you have all actual settings that work for me. Greetings, Luk Verzonden vanaf mijn Galaxy
-------- Oorspronkelijk bericht --------Van: Navid ***@***.***> Datum: 2/11/2022  13:14  (GMT+01:00) Aan: NightscoutFoundation/xDrip ***@***.***> Cc: Lukvdv ***@***.***>, Author ***@***.***> Onderwerp: Re: [NightscoutFoundation/xDrip] Samsung S22, Dexcom one issues and solutions (Discussion #2415) 
Would you please post a screenshot of the this "sync to Samsung cloud" setting so that we can see where in the menu it is?
I know it will be in Dutch.  That's OK.

**Reply by Navid200** [reply]
> The 3 dots in first pic after 

What pic?

**Reply by Navid200** [reply]
If you don't mind, please don't use email to respond to these messages.  it will be very hard for people to follow the thread.
